pkg/componentreadiness/jiraintegrator/jiraintegrator.go

```diff
@@ -0,0 +1,524 @@
+package jiraintegrator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/andygrunwald/go-jira"
+	"github.com/openshift/sippy/pkg/api/componentreadiness"
+	crtype "github.com/openshift/sippy/pkg/apis/api/componentreport"
+	"github.com/openshift/sippy/pkg/apis/cache"
+	bqclient "github.com/openshift/sippy/pkg/bigquery"
+	"github.com/openshift/sippy/pkg/db/query"
+	log "github.com/sirupsen/logrus"
+	"github.com/trivago/tgo/tcontainer"
+	jirautil "sigs.k8s.io/prow/pkg/jira"
+)
+
+const (
+	jiraProjectKey                             = "OCPBUGS"
+	jiraCreator                                = "'kenzhang@redhat.com'"
+	jiraLabelIngregration                      = "Regression"
+	jiraCustomFieldReleaseBlockerName          = "customfield_12319743"
+	jiraCustomFieldReleaseBlockerValueApproved = "Approved"
+
+	jiraStatusNew        = "New"
+	jiraStatusInProgress = "In Progress"
+	jiraStatusAssigned   = "Assigned"
+	jiraStatusModified   = "Modified"
+	jiraStatusQA         = "QA"
+	jiraStatusVerified   = "Verified"
+	jiraStatusClosed     = "Closed"
```

Comment:
Suggest moving out jira stuff like this to our own Jira package, like in pkg/apis/jira/v1. We use these values in multiple places, like pkg/dataloader/jiraloader/jiraloader.go.  Suggest matching the case to what's actually used in Jira.